KinSDK/KinSDKTests/KinAccountTests.swift

```diff
@@ -38,12 +42,11 @@ class KinAccountTests: XCTestCase {
             let publicAddress = account?.publicAddress
 
             XCTAssertEqual(publicAddress, expectedPublicAddress)
-        }
-        catch {
+        } catch {
```

Comment:
If we can't tweak this rule, I will not accept this PR.

Reply:
Of course we can. But what’s the problem?

Reply:
I prefer `catch` to be on a separate line.

Reply:
I don't really care, it's more important that we agree and stick to something, than "to be right".
It should be aligned to `if ... else` being on the same line as the closing braces. 
[Check here](https://github.com/realm/SwiftLint/blob/master/Rules.md#statement-position)

I was also going to suggest we follow the snippet that Xcode suggests. This is how it looks:

<img width="467" alt="Swift do-catch snippet" src="https://user-images.githubusercontent.com/1164565/33148510-e8976e9c-cfd4-11e7-8c7c-aa4fb7522239.png">

Reply:
I don't like tools that do not let me choose the formatting.  This is my-way-or-anything-goes, and that's stupid.

Reply:
It’s not, we can customize it